content/ru/data-center.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Дата-центр
+status: Completed
+category: Technology
+tags: ["infrastructure", "fundamental", ""]
+---
+
+Дата-центр (центр (хранения и) обработки данных, Ц(Х)ОД) - здание или объект, служащий для размещения вычислительной техники, в первую очередь - серверов.
+Часто дата-центры подключаются к высокоскоростным линиям связи с доступом к сети Интернет, особенно в том случае, если они ориентированы на [облачные вычисления](/cloud-computing/).
+Объекты, на которых размещаются дата-центры, включают необходимую инфраструктуру для поддержания работоспособности установленного на них оборудования, в том числе электрогенераторы и 
+системы вентиляции и кондиционирования, обеспечивающие охлаждение устройств с высоким выделением тепла.
+
+## Какую проблему решает
+
+До того момента, как в конце 1990-х годов дата-центры получили широкое распространение, компьютеры, в основном, использовались для решения конкретных задач или обслуживания отдельных пользователей.
+Ввиду того, что каждый компьютер располагает ограниченными ресурсами (ёмкость жёсткого диска и оперативной памяти, частота и количество ядер процессора), ограничиваются также и его возможности 
```

Comment:
Personally, I like the clarifications regarding the mentioned resources (e.g., «ёмкость жёсткого диска» instead of just «жёсткий диск»). However, I think we should follow the original version more strictly. Therefore, if we want these clarifications in our localised version, we'll need to propose this change in the English document first.

Reply:
Well, in the current version I will bring it in line with the English version. I will propose changes to the English-language document later.

Reply:
https://github.com/cncf/glossary/pull/3062/commits/59ad7906c83df92e28aa537235eb636f30604be3